configuration/mobile/android.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Mobile Android configuration"
```

Comment:
What about `Android configuration` as the title and then `Preparing Rownd to work with your Android app` as the description? (or something like that)

Reply:
Updated